src/views/customer/CustomerListView/index.tsx

```diff
@@ -14,16 +14,35 @@ const useStyles = makeStyles((theme: any) => ({
   }
 }));
 
+const client =  createClient({
+  url: 'https://iteria-app-example01.herokuapp.com/v1/graphql',
+  fetchOptions: () => {
+    return {
+      headers: {
+        'x-hasura-admin-secret' : 'iteria-app-example01-421769',
```

Comment:
never oublish secrets to opensource (in general use env variables)
in this particular case we would like to an anonymous call (hasura role anonymous in permissions tab)